docs/diagrams/webhooks-flow

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2023-02-21T08:53:46.934Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.2 Safari/605.1.15" etag="sQzrhlY7Ju5vaqSJZErk" version="20.8.20" type="device"><diagram name="Page-1" id="j61EQ_yQ_Mo7Y-l2gbru">7Vxbc5s4FP41nn1KBiHA+DF2ku7OtNlMsrPb9A2DbCsLyAXR2P31K4GEEcKxU2MgO8k0HXS46pzzncuHyAjOos2nxFuvvpAAhSPTCDYjeD0yTQBMc8T/GcG2kDgQFoJlggNx0E7wiH8iITSENMMBSpUDKSEhxWtV6JM4Rj5VZF6SkBf1sAUJ1buuvSXSBI++F+rSf3BAV4XUNcc7+e8IL1fyzsCZFHsiTx4sZpKuvIC8VETwZgRnCSG02Io2MxRy5Um9FOfd7tlbPliCYnrMCRfg2/Zimt26X+6ns+wK2PH36EJYJ6VbOWEUsPmLIUnoiixJ7IU3O+k0IVkcIH5Vg412x3wmZM2EgAmfEaVbYUwvo4SJVjQKxV60wfQrP/3SFqMnZXS9EdfOB9vK4B4lOEIUJVIW02T7tTqoXIkPd5fKR/Jauu6EOlOSJT56RWHSB71kiegrxwkv59qs3EBY5hMibBbJlh2QoNCj+IfqbZ5w2mV5XHnqPcHskU1DAAxKnAh4Aelt8hLFg4qzdt7BNiqPsRPlPvMG/7GKu/3wwkxM4QH5iM0n5c/Afv9CKX/cB/Q9y7cavO2zN2dBQ/EQL8TLmG37zD7c2NMfKKGYofJK7IhwEBTOiFL805vn1+OWXfOZ5nO3pyP7utHWryGB3whtRg2hRdxEQa9iSnGWcWlCC6pWKUa/amt5CFksUnQWMwLNjDMvDnDgUaSZjMWxNd/MovDKp6RqndyS9yTFFBNupTmhlEQN5qM8UlQNTjIa4hjNyij+Kko1I+01BrBUfEzE8GUXy6EQrSph3DH2G0fR/VsV7fQZb8Fbom3nkRUeGVnttiPrSQZ1NeQ88dg32DDntBXmLlicU6AlI+iAo5ytGeuODKcAAkcCcofBJwWCfQESON3UOpahxnLT6rbWgZr3XCOfZy1e69AVymdYlDnwVnOrZEWiecaedPqywhQ9rr1c+S+sbVKdo4WsB21VU2Nby3rAaEh7k3OlPR1464R1WQlCcVloCGUGuuZKuIHD2lvgMJyRkJcl7FwY2MgNLCZPaUL+RZU9rjmHjtOOvu1alcGUe6mrvOwEqyqHkzOpHIw/gt0bgp0MYocbO6OfaGfVa9IzRzvprEdUNh/l61scCEwGVcDK567Y+ZHNgfKkVkTooWWzkhvsLZ2VTjH4QHqCP0+O9GfT7CYgOjU3gHXznjkgSoVUgOKTaO3FW85xPaI0xSTOofO/qGEY7HqvYUz4XnDWfcEiYXcYn1Y/+LTcjgsWyeBWElmRv9iJPJstcIzTFQengTDr1xK2MecT93dcp0FKKcURF4Q4wjpvrTrZ+XOg7dpq8HMacmATkzk+Ww60P7B5CHKHsdl6LXgcNstxV9h8bzSp9O52eFJouIr6h0+VmnpX0C97MOzm73jAj7sB/LgGeNuyuwW8/mL4PiH8dSF3hn77SQfsYVYqudRtSKX1eqa9VKpTdXpncVtWL++utRg7WmXYf2uhd3MlJf2A0iykf6MEL1iW4e+y7xGrEOPlO9R9va1zGsiTbjUP9cUGZZ2ONmucNPj4R2VZY58P09QdsTJ1mrr0rwOJpstiBupU9h+LkXxnV2kS499o7neCwxF9YH4UCwC8O9ydlrtqHhzYoGgeZ+VOP8nFz2SeIy1/xMK3+X36bychHFo7Kf21KR7f7IkKw4++JQU9nOjb6wqk01d8nhA6j13PAIe1wgjq6w5mJGCN7pJZgMmvsgDzZXh1s77rdXquoQOl04V6cDwcnAyuCLGPRVJH9BaExqVhjR0wkf8rzjTpmImGOttVNBN7KtthUF7S49uhvExHXQItlxgPl/KSEeiD3zoB8VZHbYddK6xcc3I5qYaATgFv6S3tA3pmiBk04KXDt8Rx11qKUwEvLwPVM87YJeqLFAVlmTM/XE0lH9Q3hVlykXKBp9v3Ck94JJ/2Pts4W/XtScNnJMBuUHd9hXJr6pZxdr+67wjX+AK3rfGF6yPfb9L43LUtu6V+wKnRSm5T49zUELSh8dXDdD6b3l2svm228E8jtqbWRb9fSqrVAaz9vLmJPq2AqOX2gxVFozobvqNsPM46T/lQcy8AJqx+sAAYO9bYsF23lnXaqx8aJ9nQ4SfIo+VHlLT4iDIt3gT1V1IcXSmoPrEfT69VFMCAtRbCHG5F8ZqPK8RN/lqPCWchRrFOBHdN2jRYVEPz0aQNAG53rM3+aHHgRWoBrZ7KkJP0bdc/9m7SN2h6df0LWZENd3+boEDE7i88wJv/AA==</diagram></mxfile>
```

Comment:
This is the diagram in xml exported from draw.io to be able to edit it in the future, I can exclude it, but I feel like it would make much easier to modify it.

Reply:
Oh in this case I agree, it would be nice to keep diagram! Could you also update the README to explain how the diagram can be updated?